l10n_ch_invoice_reports/models/res_company.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    print_qr_invoice = fields.Boolean(
+        string="Print invoice with QR bill", readonly=False,
+    )
+    print_isr_invoice = fields.Boolean(
+        string="Print invoice with ISR payslip", readonly=False,
+    )
```

Comment:
```suggestion
    print_qr_invoice = fields.Boolean(string="Print invoice with QR bill")
    print_isr_invoice = fields.Boolean(string="Print invoice with ISR payslip")
```

by default, fields aren't readonly